src/Symfony/Component/DependencyInjection/Loader/schema/dic/services/services-1.0.xsd

```diff
@@ -302,7 +303,7 @@
     <xsd:attribute name="default-index-method" type="xsd:string" />
     <xsd:attribute name="default-priority-method" type="xsd:string" />
     <xsd:attribute name="exclude" type="xsd:string" />
-    <xsd:attribute name="exclude-self" type="xsd:boolean" />
+    <xsd:attribute name="exclude-self" type="boolean" />
```

Comment:
can you also fix the call to `getAttribute('exclude-self')` in the same PR please, wrapping it with XmlUtils::phpize?

Reply:
Ah yes sorry, I pushed a bit early the change :smile: Just pushed the call as well!